src/api/todos.ts

```diff
@@ -0,0 +1,20 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const USER_ID = 1873;
+
+export const getTodos = () => {
+  return client.get<Todo[]>(`/todos?userId=${USER_ID}`);
+};
+
+export const postTodos = ({ userId, title, completed }: Omit<Todo, 'id'>) => {
```

Comment:
```suggestion
export const addTodo = ({ userId, title, completed }: Omit<Todo, 'id'>) => {
```
you can name function depending on what does it do. It shouldn't reflect a request method in the name